mscore/preferences.cpp

```diff
@@ -158,6 +158,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_UI_CANVAS_FG_USECOLOR_IN_PALETTES,               new BoolPreference(false, false)},
             {PREF_UI_CANVAS_BG_COLOR,                              new ColorPreference(QColor("#142433"), false)},
             {PREF_UI_CANVAS_FG_COLOR,                              new ColorPreference(QColor("#f9f9f9"), false)},
```

Comment:
I guess we'd need to interpret this color as white?

Reply:
Maybe not, but why then are the vtests show background color artifacts?

Reply:
vtests are happy now